include/vcpkg/base/messages.h

```diff
@@ -1613,6 +1669,20 @@ namespace vcpkg
                     "",
                     "dependency {spec} was expected to be at least version "
                     "{expected_version}, but is currently {actual_version}.");
+    DECLARE_MESSAGE(VersionDateHelp, (), "", "A date (2021-01-01.5)");
+    DECLARE_MESSAGE(VersionFourFlavors, (), "", "Versions in vcpkg come in four primary flavors");
+    DECLARE_MESSAGE(VersionGreaterHelp,
+                    (),
+                    "",
+                    "Within the \"dependencies\" field, each dependency can have a minimum constraint listed. These "
+                    "minimum constraints will be used when transitively depending upon this library. A minimum "
+                    "port-version can additionally be specified with a '#' suffix.");
+    DECLARE_MESSAGE(VersionHelp, (), "", "A dot-separated sequence of numbers (1.2.3.4)");
```

Comment:
```suggestion
    DECLARE_MESSAGE(VersionHelp, (), "", "A dot-separated sequence of non-negative numbers (1.2.3.4)");
```

just to clarify

Reply:
The goal here is for this to fit within the width of a console together, I'm not sure this clarification is the right space/clarity tradeoff.

![image](https://user-images.githubusercontent.com/1544943/194166593-30202dbe-f292-4e35-897b-9a009bf3d69b.png)